kafkakewl-utils/src/main/scala/com/mwam/kafkakewl/utils/logging/Logging.scala

```diff
@@ -0,0 +1,72 @@
+/*
+ * SPDX-FileCopyrightText: 2023 Marshall Wace <opensource@mwam.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.mwam.kafkakewl.utils.logging
+
+import zio.*
+import zio.logging.LogFormat.{quoted, space, *}
+import zio.{Config, Runtime, ZLayer}
+import zio.logging.slf4j.bridge.Slf4jBridge
+import zio.logging.{ConsoleLoggerConfig, FileLoggerConfig, LogColor, LogFilter, LogFormat, LoggerNameExtractor}
+
+import java.nio.file.Paths
+
+object Logging {
```

Comment:
can you add a comment on PR about exactly what functionality we are losing when going from old logger to new one?
> Logging can no longer be Filtered the same way as before. ZIO does not seem to expose a way of modifying a log's level once it has been created.

Seems like you are changing log level in code?